COPYRIGHT_AND_LICENSE

```diff
@@ -1,37 +0,0 @@
-LICENSE
```

Comment:
I recall there was a reason for this file (I think was to maintain copyright holders on legacy scripts; when the wiki is up, I'll post the link).